ccmlib/dse_node.py

```diff
@@ -180,7 +178,10 @@ def start(self,
                 node.watch_log_for_alive(self, from_mark=mark)
 
         if wait_for_binary_proto:
-            self.wait_for_binary_interface()
+            self.watch_log_for("Starting listening for CQL clients", from_mark=self.mark)
```

Comment:
It already has a mark parameter, afaik. I believe it may have been to stop checking if the socket is listening.